main/externalData/surface/index.ts

```diff
@@ -0,0 +1,268 @@
+import log from 'electron-log'
+import createPylon, { Unsubscribable } from '@framelabs/pylon-api-client'
+
+import Networks from './networks'
+import bProcessor from '../balances/processor'
+import iProcessor from '../inventory/processor'
+import { TokenBalance } from '../balances/scan'
+import { formatUnits } from 'ethers/lib/utils'
+
+type Subscription = Unsubscribable & { unsubscribables: Unsubscribable[]; collectionItems: CollectionItem[] }
```

Comment:
@Jamchello a little confused about this type. its a top level subscription and then it also has an array of subscriptions?